src/utils/render-outro-message.ts

```diff
@@ -1,5 +1,5 @@
-import chalk from "chalk";
 import type { Options } from "../types";
+import chalk from "chalk";
```

Comment:
We need a prettier/linter in the CLI project :D

This was automatic for me since It's still pointing to the prettier/lint/config of SE-2